### DISCUSSION TITLE
invite to slack?

### DISCUSSION THREAD

**Discussion by coofzilla** [TheWidlarzGroup/react-native-video#3475] [General]
I saw that there was a react-native-video slack in the communications tab; but, how can I get an invite to it so that I can access it?

**Comment by KrzysztofMoch** [comment]
Here is link https://join.slack.com/t/video-dev/shared_invite/zt-24kgmctuv-2z0O9J_v6q_rg~x1RujdOA - there is `react-native-video` chanel - feel free to ask questions
(Let me know if you had any problems with slack)
